generators/generator-adaptive-bot/generators/app/templates/assets/appsettings.json

```diff
@@ -1,7 +1,29 @@
 {
   "MicrosoftAppPassword": "",
   "MicrosoftAppId": "",
-  "applicationInsights": {
-    "InstrumentationKey": ""
+  "defaultLocale": "en-US",
+  "runtimeSettings": {
+    "features": {
+      "removeRecipientMentions": false,
+      "showTyping": false,
+      "useInspection": false,
+      "traceTranscript": false
+    },
+    "telemetry": {
+      "instrumentationKey": "",
+      "logPersonalInformation": false,
+      "logActivities": true
+    },
+    "plugins": [
+      {
```

Comment:
I saw that we had a pluginDefinitions argument in the index.json but it didn't seem to be picked up anywhere. Doing this makes things just work, but wanted to check what was the original plan for the[ pluginDefinitions defined here](https://github.com/microsoft/botframework-components/search?q=plugindefinitions). Thoughts?

Reply:
There are updates that still need to be checked in. I have an open PR out for these: https://github.com/microsoft/botframework-components/pull/581

Reply:
Ahh that makes sense. So maybe I will remove them from the appsettings.json so the pluginsDefinition injects that setting which seems better.

Reply:
discussed offline and removed plugin info